proto/storage/vulnerability.proto

```diff
@@ -55,6 +55,7 @@ message EmbeddedVulnerability {
   VulnerabilityState state = 20; // @gotags: search:"Vulnerability State"
   // cvss_metrics stores list of cvss scores from different sources like nvd, Redhat etc
   repeated CVSSScore cvss_metrics = 21;
+  float nvd_cvss = 22; // @gotags: search:"NVD CVSS,store"
```

Comment:
Does it not need a policy tag too? Or is search tag enough?

Reply:
@charmik-redhat  cvss only has a search tag. based on testing so far search tag looks enough